browser.js

```diff
@@ -9,6 +9,19 @@ function wrap (down, parentOpts) {
   var levelup = packager(down)
 
   return function (loc, opts, cb) {
+    if (typeof loc === 'function') { // (cb)
+      cb = loc
+      loc = null
+      opts = null
+    } else if (typeof loc === 'object' && loc !== null) { // (opts[, cb])
+      cb = opts
+      opts = loc
+      loc = null
+    } else if (typeof opts === 'function') { // (name, cb)
+      cb = opts
+      opts = null
+    }
```

Comment:
Can we somehow make this into a function? I was thinking we could do something like

```js
[ cb, loc, opts ] = argify(loc, opts, cb)
```

Reply:
We don't support node 4 here, so that could work.

Reply:
Well, we currently support node 4.

Reply:
Oh, it's not in travis.

Reply:
#68